cypress/features/regression/actionPopoverIFrame.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component iframe
+
+  @positive
+  Scenario: Open Action Popover element
+    When I open Action Popover element
+    Then Action Popover element is visible
+
+  @positive
+  Scenario Outline: Open Action Popover element and check <innerText> as inner context
+    When I open Action Popover element
+      And I press keyboard "downarrow" key times <times>
+    Then Action Popover element "<index>" inner context is set to "<innerText>"
+    Examples:
+      | times | index | innerText     |
+      | 0     | 1     | Email Invoice |
+      | 1     | 3     | Download PDF  |
+      | 2     | 4     | Download CSV  |
+      | 3     | 6     | Delete        |
+
+  @positive
+  Scenario Outline: Open Action Popover using different keyboard key <key>
+    When I press keyboard "<key>" key times 1 on actionPopover open icon
+    Then Action Popover element "<index>" inner context is set to "<innerText>"
+    Examples:
+      | key       | index | innerText     |
+      | enter     | 1     | Email Invoice |
+      | downarrow | 1     | Email Invoice |
+      | space     | 1     | Email Invoice |
+
+  @positive
+  Scenario: Open Action Popover and close it using Home key
+    When I open Action Popover element
+      And I hit Home on Action Popover element in iFrame
+    Then  Action Popover element "1" inner context is set to "Email Invoice"
+
+  @positive
+  Scenario: Open Action Popover and close it using Uparrow key
+    When I open Action Popover element
+      And I hit uparrow on Action Popover element in iFrame
+    Then  Action Popover element "6" inner context is set to "Delete"
+
+  @positive
+  Scenario: Open Action Popover and close it using End key
```

Comment:
Change the name of the scenario, you don't close action popover

Reply:
fixed